### ISSUE TITLE
REST API: Reports wrong status for some jobs

### ISSUE THREAD

**Issue by jb68** [fossology/fossology#1591]
### Description
Using API to get the status of the jobs will show the same status for all jobs while the web interface may show them as completed.

```
curl -k -s -X GET 'http://xxxxx/repo/api/v1/jobs?upload=1179' -H 'Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGci...' -H 'accept: application/json' | jq
[
  {
    "id": 9282,
    "name": "wget",
    "queueDate": "2020-01-30 00:11:27.216465+00",
    "uploadId": "1179",
    "userId": "3",
    "groupId": "3",
    "eta": 0,
    "status": "Processing"
  },
  {
    "id": 9283,
    "name": "20.01.0.Th.rc3",
    "queueDate": "2020-01-30 00:11:27.605805+00",
    "uploadId": "1179",
    "userId": "3",
    "groupId": "3",
    "eta": 0,
    "status": "Processing"
  }
]
```

This query will return "Processing" for both until job "20.01.0.Th.rc3" even the job "wget" ended long time ago.
Expected result should be"
```
 {
    "id": 9282,
    "name": "wget",
    "queueDate": "2020-01-30 00:11:27.216465+00",
    "uploadId": "1179",
    "userId": "3",
    "groupId": "3",
    "eta": 0,
    "status": "Completed"
  },
  {
    "id": 9283,
    "name": "20.01.0.Th.rc3",
    "queueDate": "2020-01-30 00:11:27.605805+00",
    "uploadId": "1179",
    "userId": "3",
    "groupId": "3",
    "eta": 0,
    "status": "Processing"
  }
]
```

#### How to reproduce
Start a few longer jobs that depend on the same upload and monitor them with 
repo/api/v1/jobs?upload=xxx

#### Screenshots

I might be able to add screenshots as well but I think the bug is well defined. 

### Versions

* 3.7
* Fossology official docker

**Comment by GMishx** [comment]
Should be fixed by #1812

**Comment by GMishx** [comment]
Closing since merge of #1812
